### PR TITLE
Single source logic for generating invalid DVR error

### PR DESCRIPTION
**Pull request by jasonmreding**
- [x] This contribution adheres to [CONTRIBUTING.md](https://github.com/ni/measurement-services-labview/blob/main/CONTRIBUTING.md).

### What does this Pull Request accomplish?

- Single sources logic for generating an invalid DVR error to an inlined sub VI
- Updates menu name in `Get Session Info` poly VI from "Multiple Session" to "Multiple Sessions" for consistency

### Why should this Pull Request be merged?

Improves maintainability and consistency of the code.

### What testing has been done?

Relying on existing regression tests since this is a simple refactor.

### CODE REVIEW

File: Source/Runtime/MeasurementLink Session Management/Session Reservation/Get Session Info.vi
Comment (jasonmreding) [inline Source/Runtime/MeasurementLink Session Management/Session Reservation/Get Session Info.vi:1]: image

File: Source/Runtime/MeasurementLink Session Management/Session Reservation/Merge Invalid DVR Error.vi
Comment (jasonmreding) [inline Source/Runtime/MeasurementLink Session Management/Session Reservation/Merge Invalid DVR Error.vi:1]: image

File: Source/Runtime/MeasurementLink Session Management/Session Reservation/Close Sessions.vi
Comment (jasonmreding) [inline Source/Runtime/MeasurementLink Session Management/Session Reservation/Close Sessions.vi:1]: Example before and after with the change. All of the modified VIs have similar changes.

image


image

**Review by dixonjoel — APPROVED** [review]

File: Source/Runtime/MeasurementLink Session Management/Session Reservation/Merge Invalid DVR Error.vi
Comment (pbirkhol-ni) [reply Source/Runtime/MeasurementLink Session Management/Session Reservation/Merge Invalid DVR Error.vi:1]: We have a user story to start using the LabVIEW Error Ring, which would make this change unnecessary. However, it is okay for now.

**Pull request merged**